### DISCUSSION TITLE
Pipeline <class 'diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline'> expected {'vae', 'unet', 'image_encoder', 'text_encoder', 'feature_extractor', 'tokenizer', 'safety_checker', 'scheduler'}, but only {'vae', 'scheduler', 'tokenizer', 'text_encoder'} were passed.

### DISCUSSION THREAD

**Discussion by zhaoyuchen1128** [huggingface/diffusers#8717] [General]
StableDiffusionPipeline确实需要这几个参数，但是它们分别代表什么？需要什么类型？求帮助

**Comment by asomoza** [comment]
Can you please post your question in english? Also we need a a simple code snippet of what you're using so we can help. 

The error is saying that you're missing the unet.

**Reply by zhaoyuchen1128** [reply]
> Can you please post your question in english? Also we need a a simple code snippet of what you're using so we can help.
> 
> The error is saying that you're missing the unet.

I am sorry,I have made some mistakes.I will check it and reply later
